src/main/java/teammates/ui/controller/InstructorHomeCourseAjaxPageData.java

```diff
@@ -142,7 +142,9 @@ private void addAttributeIf(boolean shouldAdd, ElementTag button, String key, St
                     getInstructorHoverMessageForFeedbackSession(session),
                     getInstructorStatusForFeedbackSession(session),
                     TimeHelper.formatDateTimeForInstructorHomePage(session.getStartTime()),
+                    session.getStartTimeString(),
```

Comment:
@rclakmal I think this actually comes from a commit in #5760, let's remove commits for other PRs from this branch so the PRs can be reviewed separately

You may wish to put either PR on hold while finishing the other.